### DISCUSSION TITLE
百度贴吧大几率会卡死。

### DISCUSSION THREAD

**Discussion by sunbroke555** [blackmatrix7/ios_rule_script#664] [Q&A]
大概10分钟内必卡，就是页面卡住滑动没反应
系统ios15.4.1，手机型号iPhone12，目前就用了一个去广告分流➕一个知乎助手。
![image](https://user-images.githubusercontent.com/74763419/183555256-ede5cc84-a7cb-480e-9e0a-3f632de9436f.png)
![image](https://user-images.githubusercontent.com/74763419/183555441-71fdf45b-6421-4b53-adaa-24e65bdaa1cb.png)
![image](https://user-images.githubusercontent.com/74763419/183555451-2057fef4-316a-479b-8515-f78b0dfe9ebb.png)

**Comment by blackmatrix7** [comment]
已知问题，目前还没找到原因。
